app/hunter/reverb_api/price_guide_transactions_result.rb

```diff
@@ -0,0 +1,55 @@
+module Hunter
+  module ReverbAPI
+    class PriceGuideTransactionsResult
+      def initialize(response:, filters: [])
+        @response = response
+        @filters = Array(filters)
+      end
+
+      def total_transactions
+        parsed_results["total"]
+      end
+
+      def transactions
+        filtered_transactions
+      end
+
+      def after_date(date:)
+        filters << lambda { |transaction| transaction["date"] > date }
```

Comment:
this method of filtering feels ineloquent (add procs to a filters array, run `filtered_transactions` instance method once filters are built up) .. will look into a better way